components/integration/base/kustomization.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  # Core manifests
+  - https://github.com/konflux-ci/konflux-ci/konflux-ci/integration/core?ref=df1ab193b2b9f6224a4c09a5a2ee1bc0f3a4ae57
+  # CertManager manifests
+  - https://github.com/konflux-ci/konflux-ci/konflux-ci/integration/certmanager?ref=df1ab193b2b9f6224a4c09a5a2ee1bc0f3a4ae57
```

```suggestion
  - https://github.com/konflux-ci/konflux-ci/konflux-ci/integration?ref=f82932c62b46236f66d16044c8f893ed4b28c034
```
Refer to the root kustomization which already contains all the resources